.devcontainer/devcontainer.json

```diff
@@ -27,11 +27,17 @@
                         "path": "/bin/bash",
                         "icon": "terminal-bash"
                     }
+                },
+                "python.analysis.diagnosticSeverityOverrides": {
+                    "reportMissingImports": "none",
+                    "reportMissingModuleSource": "none",
+                    "reportUndefinedVariable": "none"
                 }
             },
             "extensions": [
                 "ms-vscode.cpptools",
                 "ms-python.python",
+                "ms-python.black-formatter",
```

Comment:
Why do we remove this here? Black is used to format python files, shouldn't we have the associated extension?

Reply:
@Derollez I'm not removing it, I'm adding black to the dev container, it didn't exist before.